hathor/cli/run_node.py

```diff
@@ -644,18 +644,18 @@ def check_unsafe_arguments(self, args: Namespace) -> None:
                 sys.exit(-1)
 
     def check_python_version(self) -> None:
-        MIN_STABLE = (3, 8)
+        MIN_VER = (3, 8)
         RECOMMENDED_VER = (3, 9)
         cur = sys.version_info
-        min_pretty = '.'.join(map(str, MIN_STABLE))
+        min_pretty = '.'.join(map(str, MIN_VER))
         cur_pretty = '.'.join(map(str, cur))
         recommended_pretty = '.'.join(map(str, RECOMMENDED_VER))
-        if cur < MIN_STABLE:
+        if cur < MIN_VER:
```

Comment:
Should we exit here? The python version is not supported

Reply:
Yeah, that'd be better than just warning.

Reply:
Done.